src/psyclone/dynamo0p3.py

```diff
@@ -6528,13 +6525,13 @@ def gen_stub(self):
     def incremented_arg(self):
         ''' Returns the argument corresponding to a field or operator that has
         INC access.  '''
-        return Kern.incremented_arg(self, FIELD_ACCESS_MAP)
+        return Kern.incremented_arg(self)
 
     @property
     def written_arg(self):
         ''' Returns the argument corresponding to a field or operator that has
         WRITE access '''
-        return Kern.written_arg(self, FIELD_ACCESS_MAP)
+        return Kern.written_arg(self)
```

Comment:
This line is not covered but, since all we're doing is calling the `written_arg` method of the parent class, this method can be removed altogether. In fact, the same applies for `incremented_arg` above.

Reply:
Done (except that those functions here had @property, which psyGen did not have ... but since I don't like @property anyway, I've added '()' to all accesses of the former property).